app/services/permission_service.rb

```diff
@@ -30,6 +30,7 @@ def allow?(controller, action)
       return true if controller == 'users'            && action.in?(%w(show edit update profile))
       return true if controller == 'stores/dashboard' && action == 'show'
       return true if controller == 'order_items'      && action == 'update'
+      return true if controller == 'stores'           && action.in?(%w(index show edit update))
```

Comment:
Line is too long. [95/80]<br>Operator `&&` should be surrounded with a single space.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.